### PR TITLE
[WIP] add a PR template for enhancements

### PR DESCRIPTION
**Pull request by kikisdeliveryservice**
Adding a first pass of an enhancements PR template.

### CODE REVIEW

File: enhancements/.github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!-- 
+	Please fill out the following questions when adding/updating an enhancement in the openshift/enhancements repo
+-->
+### Enhancement Description
+
+<!-- short description of enhancement-->  
+- Brief (1-2 lines) enhancement description:
+
+<!-- link to related discussions about this enhancement, for example a github issue, RFE or public mailing list link if available -->
```
Comment (dhellmann) [inline enhancements/.github/pull_request_template.md:9]: This is redundant with the template metadata header. Do you think we need it in both places?

File: enhancements/.github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+- Discussion links:
+
+<!-- add which team will be primarily responsible for owning and maintaining this enhancement -->
+- Primary team:
+
+<!-- add any other teams that must review or will be impacted by this enhancement -->
```
Comment (dhellmann) [inline enhancements/.github/pull_request_template.md:15]: This is definitely redundant. Is the point to list people here so they are pinged?

File: enhancements/.github/pull_request_template.md (hunk 1 above)
Comment (kikisdeliveryservice) [reply enhancements/.github/pull_request_template.md:9]: I think PRs having a standard format with a brief description is always good practice.

File: enhancements/.github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+-->
+### Enhancement Description
+
+<!-- short description of enhancement-->  
+- Brief (1-2 lines) enhancement description:
+
+<!-- link to related discussions about this enhancement, for example a github issue, RFE or public mailing list link if available -->
+- Discussion links:
+
+<!-- add which team will be primarily responsible for owning and maintaining this enhancement -->
```
[the remark this answers is not in the document]
Comment (kikisdeliveryservice) [reply enhancements/.github/pull_request_template.md:12]: I don't see why _not_ to put it in there esp to make sure things aren't just randomly opened in this repo

File: enhancements/.github/pull_request_template.md (hunk 1 above)
Comment (dhellmann) [reply enhancements/.github/pull_request_template.md:9]: Sorry, I chose a bad line for that comment. I meant the discussion links are similar to the tracking link in the metadata.

File: enhancements/.github/pull_request_template.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (dhellmann) [reply enhancements/.github/pull_request_template.md:12]: Do we have GitHub teams for all of the relevant teams? Would labels be a better way for teams to find PRs in their area?

File: enhancements/.github/pull_request_template.md (hunk 2 above)
Comment (dhellmann) [reply enhancements/.github/pull_request_template.md:15]: OK, what if we say that we want the approvers and initial reviewers listed here, then?

**Comment by dhellmann** [comment]
@kikisdeliveryservice are you still working on this?

**Pull request closed without merging**